### PR TITLE
Polus cellpose inference plugin

### PR DESCRIPTION
**Pull request by pbsudharsan**
This pr adds   segmentation of cells/nuclei using pretrained models and neural network created by the authors of Cellpose.  The plugin lets you input custom trained models as well.

Details and citation for the source are included in the README

### CODE REVIEW

**Review by Nicholas-Schaub — CHANGES_REQUESTED** [review]
There are a lot of things that need to be cleaned up before further review.

1. There are a number of extraneous xml.iml files that should not be included.
2. It's unclear to me why you use the bfio base container if you're just installing a newer version of bfio, when you could just use the pytorch container and install bfio.
3. It's unclear to me why you have copied files from cellpose rather than just installing the cellpose package and use that directly. In the case of the label to vector and vector to label plugin, it made sense to do something like this because that code is a small part of the cellpose package, and we could avoid downloading the cellpose models and avoid dependency issues. Here, it doesn't make sense because now it will be more difficult to keep our code up to date with Cellpose.
4. The overlap size of 512 seems to be too large.
5. There are a number of grammatical/labeling errors throughout the code. Double spaces, incorrect labeling of things (such as the repository tag in plugin.json).
